_apidocs/get-opportunities-public-api.md

```diff
@@ -709,6 +719,9 @@ Note: Response for one record is provided as an example <br>
       "typeOfSetAside": null,
       "responseDeadLine": "2018-05-07T20:00:00-05:00",
       "naicsCode": "336350",
+	  "naicsCodes": [
```

Comment:
There is indent issue and "naicsCodes" field is not lined up well. Can you fix it?